fluent-syntax/setup.py

```diff
@@ -2,7 +2,7 @@
 
 from setuptools import setup
 
-setup(name='fluent',
+setup(name='fluent_syntax',
```

Comment:
I'd prefer to use `_` here (or `-`; practically they're the same), but unfortunately `fluent-bundle` is [already taken on PyPI](https://pypi.org/project/fluent-bundle/). I see three paths forward:

1. We go with `fluent_syntax` and find a different name for `fluent_bundle` in the same schema, e.g. `fluent_runtime`.
2. We go with `fluent_syntax` and re-use the existing `fluent` package name for `fluent_bundle`.
3. We use a different separator. According to [PEP 426](https://www.python.org/dev/peps/pep-0426/#name), periods are allowed, too. How about `fluent.syntax` here and `fluent.bundle` in #81?

Out of the three, I think I like the option no. 3 the most, but I'm open to other opinions.

Reply:
Option 3 also looks good to me. My choice of `_` was due to following the guidelines here - https://github.com/pypa/sample-namespace-packages/tree/master/pkgutil#practical-usage - but it sounds like a period could work too. We can try that - I'm concerned the combination with namespace packages might confuse something - and if there is a problem we can think again.